test/ZNSRootRegistrar.test.ts

```diff
@@ -154,6 +161,123 @@ describe("ZNSRootRegistrar", () => {
     );
   });
 
+  it("Throws if env variable is invalid", async () => {
+    try {
+      const config = await getConfig(
+        deployer,
+        zeroVault,
+        [deployer.address, governor.address],
+        [deployer.address, admin.address],
+      );
+
+      validate(config, "other");
+
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    } catch(e : any) {
+      expect(e.message).includes(INVALID_ENV_ERR);
+    }
+  });
+
+  it("Fails to validate when mocking MEOW on prod", async () => {
+    try {
+      const config = await getConfig(
+        deployer,
+        zeroVault,
+        [deployer.address, governor.address],
+        [deployer.address, admin.address],
+      );
+      // Modify the config
+      config.mockMeowToken = true;
+      validate(config, "prod");
+
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    } catch(e : any) {
+      expect(e.message).includes(NO_MOCK_PROD_ERR);
+    }
+  });
+
+  it("Fails to validate if not using the MEOW token on prod", async () => {
+    try {
+      const config = await getConfig(
+        deployer,
+        zeroVault,
+        [deployer.address, governor.address],
+        [deployer.address, admin.address],
+      );
+
+      config.mockMeowToken = false;
+      config.stakingTokenAddress = "0x123";
+
+      validate(config, "prod");
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    } catch(e : any) {
```

Comment:
`any` here can be `Error` type, no?

Reply:
Causes a TypeScript error for me `Catch clause variable type annotation must be 'any' or 'unknown' if specified.` so I'd have to override that or the `explicit-any` either way